components/.ordering

```diff
@@ -53,6 +53,7 @@ hook
 recorder-api
 recorder-process
 recorder-mocha
+recorder-jest
```

Comment:
Well, I don't make it available to the user. It is only when I assemble all the parts together that I claim `feat:`. Otherwise, it will be awkward to have multiple `fest: jest support` in the changelog.